src/components/utility/Typography/Typography.js

```diff
@@ -13,7 +13,7 @@ export const TypographyElements = {
   h1Small: "h1",
   h2: "h2",
   h3: "h3",
-  h3Large: "h3Large",
+  h3Large: "h1",
```

Comment:
This should be an `h3` rather than an `h1`, I think. It's currently used only for the names below the quotes in src/components/HomeContainer/SuccessStories/SuccessStory.jsx so semantically, I'm not sure it should be a heading vs. a paragraph, but if it's a heading, it's not the top level context.

Reply:
I will fix that